Godeps/Godeps.json

```diff
@@ -389,6 +389,958 @@
 		{
 			"ImportPath": "gopkg.in/yaml.v2",
 			"Rev": "53feefa2559fb8dfa8d81baad31be332c97d6c77"
+		},
+		{
+			"ImportPath": "k8s.io/apimachinery/pkg/api/equality",
```

Comment:
The kube-aggregator is not a library, but a binary. So vendoring `k8s.io/*` including the files is ok IMO. @deads2k wdyt?